### PR TITLE
update mosaic amount( to be realative)

### PR DESCRIPTION
**Pull request by NemTechCN**

### CODE REVIEW

File: src/common/vue/collection-record/CollectionRecordTs.ts [hunk 1]
```diff
@@ -128,51 +185,55 @@ export class CollectionRecordTs extends Vue {
...
+
+    getRelativeMosaiccByTransaction(transactionList, node) {
+        let resultList = transactionList
+        const that = this
+        if (transactionList.length == 0) {
+            that.isLoadingTransactionRecord = false
+            return
+        }
+        Promise.all(transactionList.map(async (item, index) => {
+            this.isLoadingTransactionRecord = true
```
Comment (decentraliser) [inline src/common/vue/collection-record/CollectionRecordTs.ts]: Here async await are not needed, is loading transaction record should be outside the .map, and you should add a .catch

File: src/common/vue/collection-record/CollectionRecordTs.ts [hunk 2]
```diff
@@ -128,51 +185,55 @@ export class CollectionRecordTs extends Vue {
...
+                return mosaicHex + `(${getRelativeMosaicAmount(item.amount.compact(), MosaicDivisibilityMap[mosaicHex].divisibility)})`
+            }).join(',')
+            that.transactionDetails = transaction
+            that.initDialogData(transaction)
+            that.isLoadingModalDetailsInfo = false
+        })
 
+    }
+
+    getRelativeMosaiccByTransaction(transactionList, node) {
```
Comment (decentraliser) [inline src/common/vue/collection-record/CollectionRecordTs.ts]: Typo mosaicc

File: src/common/vue/collection-record/CollectionRecordTs.ts (hunk 1 above)
Comment (NemTechCN) [reply src/common/vue/collection-record/CollectionRecordTs.ts]: can not get mosaic amount correctly with out async await

File: src/common/vue/collection-record/CollectionRecordTs.ts (hunk 2 above)
Comment (NemTechCN) [reply src/common/vue/collection-record/CollectionRecordTs.ts]: fixed

File: src/common/vue/collection-record/CollectionRecordTs.ts (hunk 1 above)
Comment (decentraliser) [reply src/common/vue/collection-record/CollectionRecordTs.ts]: OK will check it later

**Pull request merged**